stack_definition.json

```diff
@@ -261,8 +261,58 @@
       ]
     },
     {
-      "name": "4 - WatsonX SaaS services",
-      "version_locator": "1082e7d2-5e2f-0a11-a3bc-f88a8e1931fc.c237751c-d3dd-4c23-b0b1-ba726d30c8e9-global",
+      "name": "4 - Databases for Elasticsearch",
+			"version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.c5ea982b-011a-46c8-a49d-0de273f40b46-global",
+			"inputs": [
+				{
+					"name": "use_existing_resource_group",
+					"value": true
+				},
+				{
+					"name": "resource_group_name",
+					"value": "ref:../../members/1 - Account Infrastructure Base/outputs/workload_resource_group_name"
+				},
+				{
+					"name": "region",
+					"value": "ref:../../inputs/region"
+				},
+				{
+					"name": "prefix",
+					"value": "ref:../../inputs/prefix"
+				},
+				{
+					"name": "plan",
+					"value": "platinum"
+				},
+				{
+					"name": "elasticsearch_version",
+					"value": "8.12"
+				},
+				{
+					"name": "existing_kms_instance_crn",
+					"value": "ref:../../members/2a - Security Service - Key Management/outputs/kms_instance_crn"
+				},
+				{
+					"name": "kms_endpoint_type",
+					"value": "private"
+				},
+				{
+					"name": "member_host_flavor",
+					"value": "b3c.4x16.encrypted"
+				},
+        {
+					"name": "service_credential_names",
+					"value": {
+						"elastic_db_admin" : "Administrator",
+						"wxasst_db_user" : "Editor",
+						"toolchain_db_user" : "Editor"
+					}
+				}
+			]
+		},
+    {
+      "name": "5 - WatsonX SaaS services",
+      "version_locator": "1082e7d2-5e2f-0a11-a3bc-f88a8e1931fc.9932d3b1-369e-4b5c-ad39-9e10149b83b0-global",
```

Comment:
@akocbek can you confirm what version this is, as it seems pipeline hit this issue https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/issues/112

Reply:
it is `v1.3.6` version, the latest one. the one who should have a fix